specs/OracleQueries.yml

```diff
@@ -437,32 +499,12 @@ components:
         type: integer
         format: int64
       example: "10"
-    forename:
-      name: 'forename'
-      description: The officers' first name.
-      in: 'query'
-      required: false
-      schema:
-        type: string
-    surname:
-      name: 'surname'
-      description: The officers' surname.
-      in: 'query'
-      required: false
-      schema:
-        type: string
-    dateOfBirth:
-      name: 'dateOfBirth'
-      description: The officers' date of birth.
-      in: 'query'
-      required: false
+    ephemeralOfficerKey:
+      name: 'ephemeral_officer_key'
+      description: The ephemeral key for a bankrupt officer. This is a disposable key which only lasts until the next daily upload.
```

Comment:
This refers to the implementation of the DB script. This API shouldn't have knowledge of how the DB is behaving (daily upload). If we change the DB refresh behaviour, this spec will be incorrect (this is likely to be forgotten to be updated).

I would make it more generic like:
"The ephemeral key for a bankrupt officer. Note: This is a disposable key which may change when the data is refreshed"